flask_restutils/resources.py

```diff
@@ -22,6 +22,12 @@ def raw_data(self):
         return request_json()
 
     def get_queryset(self):
+        """Return the queryset that should be used for the current request.
+
+        This method can return None instead of a queryset to indicate that
+        the queryset is empty. This is equivalent to QuerySet.none() in
+        MongoEngine/Django (SQLAlchemy doesn't offer such a construct).
```

Comment:
No need to say this last sentence here (irrelevant and this repository doesn't use MongoEngine/Django).